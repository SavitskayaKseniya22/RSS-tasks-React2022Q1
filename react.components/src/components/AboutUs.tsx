import React from 'react';

export class AboutUs extends React.Component {
  constructor(props: string) {
    super(props);
  }
  render() {
    return (
      <div>
        <h1>about us</h1>
      </div>
    );
  }
}
