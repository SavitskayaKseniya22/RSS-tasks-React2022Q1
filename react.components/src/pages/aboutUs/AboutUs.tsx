import React from 'react';

export class AboutUs extends React.Component {
  constructor(props: string) {
    super(props);
  }
  render() {
    return (
      <div data-testid="about-us">
        <h1>about us</h1>
      </div>
    );
  }
}
