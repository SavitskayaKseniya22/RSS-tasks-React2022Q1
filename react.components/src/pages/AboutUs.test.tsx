import { render, screen } from '@testing-library/react';
import { AboutUs } from './AboutUs';

test('check AboutUs page appearance', () => {
  render(<AboutUs />);
  const aboutUsPage = screen.getByTestId('about-us');
  expect(aboutUsPage).toMatchSnapshot();
});
