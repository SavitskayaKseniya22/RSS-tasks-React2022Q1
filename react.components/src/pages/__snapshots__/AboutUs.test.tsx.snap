// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check AboutUs page appearance 1`] = `
<div
  data-testid="about-us"
>
  <h1>
    about us
  </h1>
</div>
`;
